lightning/src/ln/peer_handler.rs

```diff
@@ -543,7 +574,12 @@ impl<Descriptor: SocketDescriptor, CM: Deref, RM: Deref, L: Deref, CMH: Deref> P
 		SecretKey::from_slice(&Sha256::from_engine(ephemeral_hash).into_inner()).expect("You broke SHA-256!")
 	}
 
-	/// Indicates a new outbound connection has been established to a node with the given node_id.
+	/// Indicates a new outbound connection has been established to a node with the given node_id
+	/// and an optional remote network address.
```

Comment:
Note that single newlines are ignored when rendering the docs, so you have to add a blank line to get a break in the docs - may want to add one after this line.